packages/@uppy/provider-views/src/Item/components/ItemIcon.js

```diff
@@ -18,7 +18,7 @@ function FolderIcon () {
 
 function VideoIcon () {
   return (
-    <svg aria-hidden="true" focusable="false" viewBox="0 0 58 58">
+    <svg aria-hidden="true" focusable="false" viewBox="0 0 58 58" style={{ width: 16, marginRight: 4 }}>
```

Comment:
what is the reason for this change? 🤔

Reply:
the video icons weren't showing up before this so setting the width / margin like was done for FileIcon and FolderIcon fixes this. without, it looks like this (it usually isn't a problem because there's a thumbnail but zoom doesn't provide one so we need the icons to show):
<img width="430" alt="Screen Shot 2020-07-16 at 10 09 25 AM" src="https://user-images.githubusercontent.com/65177495/87681348-91749500-c74c-11ea-8b65-1ca0320aa757.png">
after: